package.json

```diff
@@ -176,7 +169,6 @@
     "style-loader": "3.3.3",
     "stylelint": "16.5.0",
     "svgo": "3.3.2",
-    "twig-html-loader": "0.1.9",
```

Comment:
If it's an error in the `uswds-site` repository, I would imagine that's an issue which ought to be resolved there, i.e. ensuring that the dependency is installed in that project?

Reply:
We should add this back in for now and create a new issue to move this work to the site repo.

IIRC it was added because we followed Fractal's (previous component library tool) method. It used to build into static HTML, which the website then took to display component code.

Reply:
Restored in 964dfcc29.

Still don't know why `uswds-site` would have anything to do with this. Downstream projects don't install `devDependencies` of `@uswds/uswds`.

Reply:
Thanks @aduth. Opened up #6082 as a follow-on issue to explore the risks of removing this dependency.